lib/taste_tester/tunnel.rb

```diff
@@ -49,9 +49,10 @@ def cmd
       else
         pid = '\\$\\$'
       end
+      @max_ping = @delta_secs / 10
       cmds = "ps -p #{pid} -o pgid | grep -v PGID" +
```

Comment:
PPID isn't the same thing as the progress group id. The goal is to get the PG id and then kill that. That requires a negation (kill -PGID)  ... perhaps we're missing that somewhere.

But you're probably right we should get the PGID of the parent...  so maybe:

```
ps -p $(cat /proc/#{pid}/status | grep PPid | cut -f2) -o pgid | pgrep -v PGID
```